app/models/preserved_copy.rb

```diff
@@ -63,4 +64,14 @@ def update_status(new_status)
   def matches_po_current_version?
     version == preserved_object.current_version
   end
+
+  # Input: Takes a Time object, String, or Nil
+  # Output: If String method will return a time object,
+  # if nil, method will return the start of the Epoch Time Object
+  # and if Time object, it will return the same Time object.
+  private_class_method def self.normalize_date(timestamp)
+    timestamp = '1970-01-01T00:00:00Z' if timestamp.nil?
+    timestamp = Time.parse(timestamp) unless timestamp.instance_of?(Time)
```

Comment:
With `timestamp = Time.parse(timestamp).utc unless timestamp.instance_of?(Time)`, all specs/cops pass, without the Rubocop exception. Is there another reason not to add `.utc`?